var/spack/repos/builtin/packages/npb/package.py

```diff
@@ -22,9 +22,19 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+import numbers
+
 from spack import *
 
 
+def is_integral(x):
```

Comment:
Perhaps `variant.py` could provide a set of helper functions for validating variant values?